.github/workflows/changelog-check.yml

```diff
@@ -2,9 +2,11 @@ name: Changelog in sync for packages
 
 on:
   push:
-    branches:
-      - master
   pull_request:
+    branches-ignore:
+      - l10n_master
+      - dependabot/*
+      - chenagelog # just testing
```

Comment:
?

Reply:
I was testing to ignore this branch.. but it doesn't seem that it's skipping it.

Reply:
maybe because it's misspelled

Reply:
Well it's the same as the current branch name, as I copy pasted it.. 😄

Reply:
My guess would be that that shouldn't run at all if it's ignored..
<img width="870" alt="Screenshot 2020-12-14 at 3 17 05 PM" src="https://user-images.githubusercontent.com/125676/102085516-6e473200-3e1f-11eb-9c7f-f4a647fade27.png">